nautobot/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nautobot-ui",
```

Comment:
Looks like `nautobot-ui` is likely to be the name of the library added by #3205... let's brainstorm what we might want to call this instead?

Reply:
Agreed. We can just call it `nautobot` since it's the compiled app and won't be published to NPM (although we should squat the name).